packages/block-editor/src/store/reducer.js

```diff
@@ -602,7 +587,9 @@ const withBlockReset = ( reducer ) => ( state, action ) => {
 		const newState = {
 			...state,
 			byClientId: getFlattenedBlocksWithoutAttributes( action.blocks ),
-			attributes: getFlattenedBlockAttributes( action.blocks ),
+			attributes: new Map(
+				Object.entries( getFlattenedBlockAttributes( action.blocks ) )
+			),
```

Comment:
This might be something we want to do in `getFlattenedBlockAttributes()` instead of wrapping every `getFlattenedBlockAttributes()` call in a `Object.entries()`. WDYT?

Reply:
yes, I do think that we might end up doing that but since I only changes the "attributes" part of the state to be a map for now, we can't refactor it yet.